compute/instances/preemptible/create_preemtible.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START compute_preemptible_create]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+// createPreemtibleInstance creates a new preemptible VM instance
+// with Debian 10 operating system.
+func createPreemtibleInstance(
+	w io.Writer,
+	projectID, zone, instanceName string,
+	preemtible bool,
```

Comment:
nit: I am not sure if this param should be an input. The way the description is worded for this sample I think that value is expected to always be true, right? If so please remove.